doc-High_Availability_Guide/topics/Renaming_HA_appliances.adoc

```diff
@@ -0,0 +1,53 @@
+[[renaming-ha]]
+== Renaming Database-Only Appliances
+
+Renaming database-only appliances in a high availability setup must be performed in a specific order to maintain database replication.
+
+. Confirm and note the hostnames of the active database (primary) appliance and the standby database appliance in the appliance console. 
++
+You can verify this from the appliance summary screen of a database-only appliance, where the status for *Local Database Server* shows _primary_ or _standby_, for example:
++
+----
+...
+Local Database Server:   running (primary)
+...
+----
++
+. Stop the failover monitor on the non-database {product-title_short} appliances:
+.. In the appliance console menu, select *Configure Application Database Failover Monitor*. 
+.. Select *Stop Database Failover Monitor*.
+. Stop the database service on the standby database-only appliance:
++
+----
+# systemctl stop rh-postgresql95-postgresql
+----
++
+. Stop `evmserverd` on each {product-title_short} (non-database) appliance:
++
+----
+# systemctl stop evmserverd
+----
++
+. Rename the database-only appliance and restart that appliance:
+.. In the appliance console, configure the new hostname or IP address from the *Configure Network* option.
+.. Restart the renamed appliance.
+//Rename, then restart the standby if desired now? [Dayle: need to confirm the goal with bug reporter.]
++
+. On all appliances, change the host key to the primary database server's new hostname in `vmdb/config/database.yml`.
+. Start `evmserverd` on each {product-title_short} (non-database) appliance:
++
+----
+# systemctl start evmserverd
+----
++
+[IMPORTANT]
+====
+If you are using non-dedicated database appliances, also stop `evmserverd` on those appliances before renaming it and reconfigure the `database.yml` before restarting.
+====
+
+After `evmserverd` has started successfully, all appliances will be able connect to the database. 
+
+After that you will have to reconfigure high availability from xref:configuring_primary_db[] 
+//(part 2.3 in Installation - All steps from 2.3 to 2.6?).
+
+
```

Comment:
I think we need another step to restart the failover monitor service on all the appliances running `evmserverd`.

Reply:
Cool, I copied over the steps from sections 2.3-2.5 for now and added a step to restart the failover monitor. Thanks!